src/goerliBot.js

```diff
@@ -79,13 +111,16 @@ utils.initializeCachedNonce();
 module.exports = {
   name: 'goerliBot',
   description: 'Sends goerli eth to the user.',
-  execute(message, args, amount, runCustomChecks = false) {
+  execute(message, args, amount, runCustomChecks = true) {
     runGoerliFaucet(message, args[1], amount, runCustomChecks);
```

Comment:
This is great place to escape `address` before using it in all the business logic later, because here it comes from outside (from user)